test/xpu_api/language.support/support.initlist/support.initlist.access/access.pass.cpp

```diff
@@ -1,26 +1,35 @@
+// -*- C++ -*-
 //===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
 //
 // template<class E> class initializer_list;
 //
 // const E* begin() const;
 // const E* end() const;
 // size_t size() const;
-//
-//===----------------------------------------------------------------------===//
 
-#include "oneapi_std_test_config.h"
-#include "test_macros.h"
-#include <CL/sycl.hpp>
-#include <iostream>
+#include "support/test_config.h"
+
+#include <oneapi/dpl/cstddef>
```

Comment:
Why do we need to include this header?
(It seems that we didn't need it before)

Reply:
In source test code this include exist too:
```C++
#ifdef USE_ONEAPI_STD
#    include _ONEAPI_STD_TEST_HEADER(cstddef)
namespace s = oneapi_cpp_ns;
#else
#    include <cstddef>
namespace s = std;
#endif
```

Reply:
Removed